integration_tests/features/steps/fal_steps.py

```diff
@@ -188,33 +188,72 @@ def check_model_results(context):
     expected_models = list(map(_script_filename, context.table.headings))
     unittest.TestCase().assertCountEqual(models, expected_models)
 
-LAYOUT_PARSER = re.compile(
-    r"\d+\."
-    r" (?P<subjects>[\w.]+(, [\w.]+)*)"
-)
 
-def _parse_ordered_layout(source) -> Iterator[Sequence[str]]:
-    for match in LAYOUT_PARSER.finditer(source):
-        yield set(map(_script_filename, match["subjects"].split(", ")))
+@then("the following models are calculated in order")
```

Comment:
this means we keep around the one that does not check order, right? `@then("the following models are calculated")`

Reply:
Yep. I initially thought about verifying everything, but then it became a bit excessive (in terms of time) and problematic on some scenarios (e.g., you might be using a different `profile_target` on the CLI etc.). So for reducing the CI time & overall complexity, this is enabled on-demand.